soccer/main.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
this is great to have to Python 2 and 3 compatibility....can you make sure other places in the code where the print statement is used is now replaced by the print function? This could be done in a separate pull request